### ISSUE TITLE
JRE Variants No Longer Supported For Base Tomcat Image

### ISSUE THREAD

**Issue by julienchastang** [Unidata/tomcat-docker#71]
See: 

https://github.com/docker-library/tomcat/issues/175

Must move to a jdk base image to obtain an up-to-date Tomcat.

**Comment by julienchastang** [comment]
This will increase the size of this container somewhat:

```
| tomcat:8.5-jre8  | 462.7MB |
| tomcat:8.5-jdk8  | 506.8MB |
```

Roughly a 10% increase in size.

**Comment by julienchastang** [comment]
closed by #73 , #74 , #75 , #76
